test/e2e/antreapolicy_test.go

```diff
@@ -784,6 +785,67 @@ func testACNPAllowNoDefaultIsolation(t *testing.T) {
 	executeTests(t, testCase)
 }
 
+// testACNPAllowNoDefaultIsolationUDP tests that no default isolation rules are created for Policies in UDP.
+func testACNPAllowNoDefaultIsolationUDP(t *testing.T) {
```

Comment:
can we avoid duplicating the code? This seems very similar to `testACNPAllowNoDefaultIsolationSCTP` and `testACNPAllowNoDefaultIsolation`

Reply:
Done.